scripts/render-configs.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+# Populates local cert files necessary for local development
+# You MUST be on the Broad VPN
+# You MUST have gcloud and kubectl installed to run this script.
+# You MUST authenticate via gcloud
+#
+# See usage section below for more details. All arguments are optional.
+
+set -eu
+set -o pipefail
+
+usage() {
+    cat <<EOF
+Usage: $0 [OPTION]...
+Generate cert files for local development
+  --project PROJECT     Google project where cert files are stored
+  --env ENV             Write an .env.local file to project root. true|false. Defaults to false
+  --config CONFIG       Write a config.json file in public. true|false. Defaults to false
+  --help                Display this help and exit
+EOF
+    exit 0
+}
+
+error() {
+    echo "ERROR: $1" >&2
+    exit 1
+}
+
+# default values that may be overridden by command line arguments
+PROJECT="broad-dsde-dev"
+ENV="false"
+CONFIG="false"
+
+parse_cli_args() {
+    while [ $# -gt 0 ]; do
+        case "$1" in
+            --project)
+                PROJECT=$2
+                shift 2
+                ;;
+            --env)
+                ENV=$2
+                shift 2
+                ;;
+            --config)
+                CONFIG=$2
+                shift 2
+                ;;
+            --help)
+                usage
+                ;;
+            *)
+                error "Unknown option: $1. Try --help to see a list of all options."
+                ;;
+        esac
+    done
+}
+
+auth_gcloud() {
+  echo "Getting cluster credentials"
+  gcloud container clusters get-credentials --zone us-central1-a --project "$PROJECT" terra-dev
+}
+
+write_certs() {
+  echo "Writing cert files"
+  kubectl -n local-dev get secrets local-dev-cert -o 'go-template={{ index .data "tls.crt" | base64decode }}' > ../server.crt
```

Comment:
This is neat.

Reply:
This is cribbed from the [leonardo setup script](https://github.com/DataBiosphere/leonardo/blob/3aabca9750e77df7c0639ae8eec0f5dbe2781345/local/depends.sh#L231)